README.md

```diff
@@ -161,6 +161,54 @@ func = autoArgs: fn: args: ...
 # /nix/store/nm5fxk0kzm3mlx1c22byfs4jizajwbk1-nixpkgs-20.09pre237349.f9f48250fe1/nixpkgs/lib/customisation.nix:117
 null
 ```
+## NixOS Installation
+
+To install the plugin system wide on NixOS, create a basic derivation for `nix-doc` called `default.nix`.
+
+```nix
+{ stdenv, rustPlatform, fetchgit, pkgs, ... }:
+
+rustPlatform.buildRustPackage rec {
+  pname = "nix-doc";
+  version = "v0.3.1";
+
+  src = fetchgit {
+    rev = version;
+    url = "https://github.com/lf-/nix-doc.git";
+    sha256 = "1hiz0fsbsl74m585llg2n359gs8i2m6jh8zdikkwxd8xq7qmw032";
+  };
+
+  buildInputs = with pkgs; [ boost nix ];
+
+  nativeBuildInputs = with pkgs; [ pkg-config ];
+
+  cargoSha256 = "1d1gvx14yai4dyz44pp2ffx2swfxnm0fwvldy96dw9gqmar69cpv";
+
+  meta = with stdenv.lib; {
+    homepage = url;
+    license = licenses.lgpl3;
+    description = "A Nix documentation lookup tool that quickly dumps the docs of the library function";
+  };
+}
+```
+
+Expose the package using the `configuration.nix` options,
```

Comment:
As a non NixOS user, I'm not sure where configuration.nix normally lives, which might be useful to add.

Reply:
Sure, tell me if the wording looks good.